api/routes/project.py

```diff
@@ -56,61 +69,71 @@ async def create_project(
 @router.get('/seqr/all', operation_id='getSeqrProjects')
 async def get_seqr_projects(connection: Connection = get_projectless_db_connection):
     """Get SM projects that should sync to seqr"""
-    ptable = ProjectPermissionsTable(connection)
-    return await ptable.get_seqr_projects()
+    projects = connection.all_projects()
```

Comment:
Could we actually return all projects, filter for seqr and check if the user has access to them all?
For example, we _always_ want to joint-call all seqr projects, and if the service-account doesn't have access to a seqr project, then we should fail here.

Reply:
Thanks, that's a great pick up, I didn't realise that is what this endpoint was used for, I've updated the code to raise a forbidden error if the user doesn't have access to list all seqr projects